dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+
+## permit_improvement_type
+
+{% docs column_permit_improvement_type %}
+Type of improvement indicated by the permit.
+
+Possible values for this variable are:
+
+- null: No improvement code assigned
+- `110 - NEW CONSTRUCTION - MAJOR IMPRVT`
+- `111 - NEW BUILDING`
+- `112 - ADDITIONS`
+- `113 - DORMERS`
+- `114 - OTHER - MAJOR NEW CONSTRUCTION`
+- `130 - NEW CONSTRUCTION - MINOR IMPRVMT`
+- `131 - SWIMING POOL - TENNIS COURT`
+- `132 - DRIVEWAYS - PATIOS - WOOD DECK`
+- `133 - FENCING (and Gates)`
+- `134 - GARAGE CARPORTS BARNS`
+- `135 - OTHER-MINOR NEW CONSTRUCTION`
+- `151 - BASEMENT ROOM - REC ROOM`
+- `152 - ATTIC ROOM - BATHROOMS`
+- `153 - FIREPLACE-CENTRAL AIR CONDITIONING`
+- `154 - OTHER - REMODELING`
+- `155 - ???`
+- `156 - REMODELING - INTERIOR ONLY`
+- `157 - REMODELING - EXTERIOR ONLY`
+- `171 - MAJOR IMPROVEMENT - WRECK`
+- `172 - MINOR IMPROVEMENT - WRECK`
+- `191 - MAJOR IMPROVEMENT - BURNOUT`
+- `192 - MINOR IMPROVEMENT - BURNOUT`
+- `211 - NEW BUILDING`
+- `212 - ADDITIONS`
+- `213 - OTHER - MAJOR NEW CONSTRUCTION`
+- `231 - SWIMMING POOL - TENNIS COURT`
+- `232 - DRIVEWAYS - PATIOS`
+- `233 - FENCING`
+- `234 - GARAGE-BARN-BUTLER BUILDING`
+- `235 - OTHER-MINOR NEW CONSTRUCTION`
+- `236 - ???`
+- `251 - PARTITIONING`
+- `252 - DECONV/CONV AMT. LIV. UT.`
+- `253 - AIR CONDITIONING - FIREPLACES`
+- `254 - BASEMENT-ATTIC ROOM BATH`
+- `255 - REPLACE-REPAIR BUIDLING - FACTORY`
+- `256 - OTHER REMODELING`
+- `257 - REMODELING - INTERIOR ONLY`
+- `258 - REMODELING - EXTERIOR ONLY`
+- `271 - MAJOR IMPROVEMENT - WRECK`
+- `272 - MINOR IMPROVEMENT - WRECK`
+- `291 - MAJOR IMPROVEMENT - BURNOUT`
+- `292 - MINOR IMPROVEMENT - BURNOUT`
+- `300 - RAILROAD PROPERTIES`
+- `310 - NEW CONST MAJOR IMPROVEMENT`
+- `311 - CARRIER - NEW MAJOR CONSTRUCTION`
+- `312 - NON-CARRIER NEW MAJOR CONSTRUCTION`
+- `330 - NEW CONSTRUCTION  MINOR IMPROVEMENT`
+- `331 - CARRIER - REMODELING`
+- `332 - NON-CARRIER-REMODELING`
+- `350 - REMODELING`
+- `351 - CARRIER - REMODELING`
+- `352 - NON-CARRIER - REMODELING`
+- `370 - WRECKS`
+- `371 - MAJOR IMPROVEMENT WRECK`
+- `372 - MINOR IMPROVEMENT - WRECK`
+- `390 - BURNOUTS`
+- `391 - MAJOR IMPROVEMENT - BURNOUT`
+- `392 - MINOR IMPROVEMENT - BURNOUT`
+- `400 - EXEMPT PROPERTIES`
+- `411 - LEASEHOLD`
+- `510 - RECHECKS`
+- `511 - OCCUPANCY`
+- `512 - PRIOR YEAR PARTIAL ASSESSMENT`
+- `515 - MANUALLY ISSUED PERMIT`
+- `516 - PRIOR YEAR RECHECK`
+- `900 - OTHER`
+{% enddocs %}
+
+## permit_job_code_primary
+
+{% docs column_permit_job_code_primary %}
+Primary job description.
+
+This field functions as a permit classification field. The
+`job_code_secondary` and `improvement_code_*` fields provide
+codes with more details on the type of job that the permit represents.
+
+Possible values for this variable are:
+
+- null: No job description assigned
+- `1: RESIDENTIAL PERMIT`
+- `2: COMMERCIAL PERMIT`
+- `3: RAILROAD PERMIT`
+- `4: EXEMPT PERMIT`
+- `5: OFFICE PERMIT`
+- `6: OCCUPANCY PERMIT`
+- `7: OTHER`
```

Comment:
For the sake of consistency:

```suggestion
- `1 - RESIDENTIAL PERMIT`
- `2 - COMMERCIAL PERMIT`
- `3 - RAILROAD PERMIT`
- `4 - EXEMPT PERMIT`
- `5 - OFFICE PERMIT`
- `6 - OCCUPANCY PERMIT`
- `7 - OTHER`
```

Reply:
Fixed in 7d94c97522a10d4303a21f3e5634ae39c567aa4b.